src/server/components/comparison.ts

```diff
@@ -112,8 +112,9 @@ function getComparisonGeneNominations(gene: Gene, teams: Team[]) {
   return data;
 }
 
-function getComparisonGene(gene: Gene, teams: Team[], scores: Scores[]) {
+function getComparisonGene(gene: Gene, teams: Team[], scores: Scores[], allBiodomains: BioDomains[] | undefined) {
   const geneScores = scores.find(score => score.ensembl_gene_id == gene.ensembl_gene_id);
+  const geneBiodomains = allBiodomains?.find(b => b.ensembl_gene_id === gene.ensembl_gene_id)?.gene_biodomains.map(b => b.biodomain);
   
```

Comment:
This is technical debt from before my time.  Ideally we would call the gene collection and wouldn't have to join the biodomains / score data as this is not taking advantage of MongoDB.